src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -54,6 +54,7 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_NO_CHANGE = "Nothing will change.";
```

Comment:
Could incorporate the word "(case-insensitive)" into the message to be more explicit, as `John Doe` equals to `john doe`.

Reply:
Modified so the error message only shows when the matching is case-sensitive.